CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ FetchContent_Declare(ios-cmake
     GIT_REPOSITORY https://github.com/leetal/ios-cmake.git
     GIT_TAG 04d91f6675dabb3c97df346a32f6184b0a7ef845)
 FetchContent_Declare(JsRuntimeHost
-    GIT_REPOSITORY https://github.com/bghgary/JsRuntimeHost.git
```

Comment:
Pulling from your fork? Is this unintentional?

Reply:
I think you are looking at an outdated commit. It should be correct in the latest commit.